dune_api_scripts/update_appdata_view.py

```diff
@@ -15,10 +17,9 @@
     AS VALUES {VALUES};"""
 
     # update query in dune
-    query_id = int(getenv('QUERY_ID_ALL_APP_DATA', "257782"))
+    query_id = int(getenv("QUERY_ID_ALL_APP_DATA", "257782"))
 
-    dune.initiate_new_query(query_id, query=QUERY)
-
-    # run query in dune
-    dune.execute_query(query_id)
+    dune_query = DuneQuery("", "", QUERY, Network.MAINNET, [], query_id)
+    # fetch data
+    dune.execute_query(dune_query)
```

Comment:
This change removed the `dune.initiate_query` and only left execute which means we were refreshing a static query without updating the values. Fix is made in #8